chatcommands.py

```diff
@@ -1162,6 +1162,18 @@ def checkpost(msg, url, alias_used='scan'):  # FIXME: Currently does not support
     url = to_protocol_relative(post_data.post_url)
     post = Post(api_response=post_data.as_dict)
 
+    if has_already_been_posted(post_data.site, post_data.post_id, post_data.title) \
+            and not is_false_positive((post_data.post_id, post_data.site)):
+        # Don't re-report if the post wasn't marked as a false positive. If it was marked as a false positive,
+        # this force scan might be attempting to correct that/fix a mistake/etc.
+
+        response_text = "This post is already recently reported"
+        if GlobalVars.metasmoke_key is not None:
+            ms_link = "https://m.erwaysoftware.com/posts/by-url?url={}".format(url)  # se_link == url
+            return response_text + " [ [MS]({}) ]".format(ms_link)
```

Comment:
These should be `raise CmdException("text.....")`

Reply:
I didn't understand that. Why should it be an "exception" like invalid URL or unable to find post?

Reply:
Whenever the user provides an ‘invalid’ input (which includes scanning a recently reported post), or the command throws an error, it’s more semantically correct to raise a command exception. It’s similar to our old `status=false` argument for the old chat system.

Reply:
"raising a CmdException will always post the response, even if the command is silent (with a -). So it's best used for errors where the invoker needs to know what they did wrong."

Reply:
@angussidney Thanks to quartata and I changed it.